recipes/sdl/all/conanfile.py

```diff
@@ -247,18 +249,84 @@ def _configure_cmake(self):
                     # FIXME: Otherwise 2.0.16 links with system wayland (from egl/system requirement)
                     cmake_extra_ldflags += ["-L{}".format(os.path.join(self.deps_cpp_info["wayland"].rootpath, it)) for it in self.deps_cpp_info["wayland"].libdirs]
                     self._cmake.definitions["WAYLAND_SHARED"] = self.options["wayland"].shared
-                    self._cmake.definitions["WAYLAND_SCANNER_1_15_FOUND"] = 1  # FIXME: Check actual build-requires version
+                    self._cmake.definitions["WAYLAND_SCANNER_1_15_FOUND"] = tools.Version(self.deps_cpp_info["wayland"].version) >= "1.15.0"
```

Comment:
```suggestion
                    self._cmake.definitions["WAYLAND_SCANNER_1_15_FOUND"] = 1  # FIXME: Check actual build-requires version
```

using `deps_cpp_info` we get the information from the _host_ context, not the _build_ one. Here we need to check the version of the build-requires, not the requires

Reply:
wayland is not build context, the comment was wrong

Reply:
I'll revert this change, didn't see that wayland was also in build requirements.